### PR TITLE
feat!: drop import sort

### PR DESCRIPTION
**Pull request by itajaja**
BREAKING CHANGE: Drop import-sort

### CODE REVIEW

**Comment by itajaja** [comment]
confused, who was using `sort-imports.js`

File: packages/init/templates/package.json.hbs [hunk 1]
```diff
@@ -76,10 +76,9 @@
   {{/unless}}
 
   "devDependencies": {
-    "@4c/cli": "^1.0.1",
-    "@4c/jest-preset": "^1.3.0",
+    "@4c/cli": "^1.1.0",
```
Comment (itajaja) [inline packages/init/templates/package.json.hbs]: chicken and egg situation :P current version is `1.0`

**Comment by itajaja** [comment]
@jquense how can I test this?

File: packages/init/templates/package.json.hbs [hunk 2]
```diff
@@ -76,12 +80,11 @@
...
+    "@4c/jest-preset": "^1.4.5",
     "@4c/prettier-config": "^1.1.0",
-    "@4c/import-sort": "^4.3.3",
     {{#if babel}}
-      "@babel/cli": "^7.2.3"
+      "@babel/cli": "^7.2.3",
```
Comment (jquense) [inline packages/init/templates/package.json.hbs]: omg thank you

Comment (itajaja) [reply packages/init/templates/package.json.hbs]: :D

**Comment by itajaja** [comment]
I tested it to the extent I could, I also did `cli init` and everything is in order

**Comment by itajaja** [comment]
@jquense not sure how to do that in practice.

**Comment by itajaja** [comment]
Feel free to rename and merge

**Comment by taion** [comment]
shoot, that was slightly wrong. i'm going to force push this away to fix

**Comment by taion** [comment]
eh actually it's nbd

**Comment by itajaja** [comment]
What was wrong?

**Comment by taion** [comment]
we don't need this marked as a breaking change for e.g. `svg2c`...

**Comment by taion** [comment]
err, what the heck, i could have sworn i did "squash and merge"

**Comment by taion** [comment]
okay, turns out i do need to re-do this

**Pull request merged**